guava/src/com/google/common/math/BigIntegerMath.java

```diff
@@ -144,8 +147,10 @@ public static int log2(BigInteger x, RoundingMode mode) {
    *     is not a power of ten
    */
   @GwtIncompatible // TODO
-  @SuppressWarnings("fallthrough")
-  public static int log10(BigInteger x, RoundingMode mode) {
+  @SuppressWarnings({"fallthrough",
+          "lowerbound:argument.type.incompatible"// BigIntegerMath.log10() only takes in positive x values
```

Comment:
This comment is confusing, because there is no call to `BigInteger.log10()` in the code below. Maybe you're referring to `LongMath.log10`. However, I don't see why a warning suppression should be necessary here - shouldn't `BigInteger#longValue` be annotated as `@PolyIndex long longValue(@PolyIndex BigInteger this);`?